qemu/tests/watchdog.py

```diff
@@ -146,7 +153,7 @@ def check_watchdog_support():
         error_context.context("Checking whether or not the host support"
                               " WDT '%s'" % watchdog_device_type, logging.info)
         watchdog_device = process.system_output("%s 2>&1" % qemu_cmd,
-                                                shell=True)
+                                                shell=True).decode()
```

Comment:
@YanhuiMa  I have changed this part via a separate patch.  and patch is merged.  so you can remove it.

Reply:
(17/31) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.watchdog.watchdog_action.inject_nmi.i6300esb: PASS (355.91 s)
 (18/31) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.watchdog.watchdog_action.inject_nmi.ib700: PASS (345.66 s)

Reply:
Acked-by Xiangchun Fu xfu@redhat.com